confetti/src/main/java/com/github/jinatonic/confetti/confetto/Confetto.java

```diff
@@ -70,6 +65,77 @@
     private float overrideX, overrideY;
     private float overrideDeltaX, overrideDeltaY;
 
+    // Visible for testing
+    protected static Long computeMillisToReachTarget(Float targetVelocity, float initialVelocity,
+                                                     float acceleration) {
+        if (targetVelocity != null) {
+            if (acceleration != 0f) {
+                final long time = (long) ((targetVelocity - initialVelocity) / acceleration);
+                return time > 0 ? time : 0;
+            } else {
+                if (targetVelocity < initialVelocity) {
+                    return 0L;
+                } else {
+                    return null;
+                }
+            }
+        } else {
+            return null;
+        }
+    }
+
+    // Visible for testing
+    protected static long computeBound(float initialPos, float velocity, float acceleration,
```

Comment:
same thing for this, please do not move methods around unnecessarily, also makes it harder to review.